src/citrine/informatics/report.py

```diff
@@ -0,0 +1,31 @@
+"""A resource that represents a single module report."""
```

Comment:
There's a `src/citrine/informatics/reports.py` file (note the s). Having both would be confusing, but I'm not clear on what the classes in `reports.py` are for

Reply:
reverted this